backend/server.ts

```diff
@@ -0,0 +1,27 @@
+import express, { Request, Response } from "express";
+import bodyParser from "body-parser";
+import cors from "cors";
+import fs from "fs";
+import dotenv from "dotenv";
+
+dotenv.config();
+const app = express();
+const port = process.env.PORT;
+
+app.use(cors());
+app.use(bodyParser.json());
+
+app.get("/api/products", (_req: Request, res: Response) => {
+    try {
+        const productsRawData = fs.readFileSync("./api/db/db.json", "utf-8");
```

Comment:
will not be better to call this file product, at least for now?

Reply:
I agree. Changed to products.json

Reply:
Agree - changed to products.json